envs/dexseq_DTU.yaml

```diff
@@ -1,4 +1,4 @@
-name: dexseq_dtu
+name: dexseq_dtu_2
```

Comment:
dexseq_dtu_2? das is vielleicht fuer debugging interessant aber nicht fuer den offiziellen branch